projects/plugins/jetpack/extensions/shared/components/product-management-controls/store/selectors.js

```diff
@@ -0,0 +1,24 @@
+export const getAllProperties = state => ( {
+	apiState: state.apiState,
+	connectUrl: state.connectUrl,
+	shouldUpgrade: state.shouldUpgrade,
+	siteSlug: state.siteSlug,
+} );
+
+export const getApiState = state => state.apiState;
+
+export const getConnectUrl = state => state.connectUrl;
+
+export const getProducts = state => state.products;
+
+export const getProductsNoResolver = state => getProducts( state );
```

Comment:
Just in the off-chance this is not clear:

The `getProducts` selector, when called through Redux, is passed through the `getProducts` resolver that fetches the products from the backend and add them to the state.

`getProductsNoResolver` (the name is consistent with [Core Gutenberg](https://github.com/WordPress/gutenberg/blob/00a7f89f80cea844e363bbe7789fde1d574ae8ad/packages/core-data/src/selectors.js#L238)) bypasses the resolver calling `getProducts` directly, which means we get to save an API call if not needed.